### PR TITLE
fix: populate `--core.ip` for keys and wallet section

### PR DESCRIPTION
**Pull request by rootulp**
Closes https://github.com/celestiaorg/docs/issues/910

It's possible these were intentionally left as placeholders in the keys and wallet section. If that's the case, the issue and PR can be closed as won't fix.

### CODE REVIEW

File: docs/nodes/light-node.mdx [hunk 1]
```diff
@@ -10,10 +10,10 @@ import blockspaceraceVersions from "../../versions/blockspacerace_versions.js";
 
 # Setting up a Celestia light node
 
-````mdx-code-block
+```mdx-code-block
```
Comment (rootulp) [inline docs/nodes/light-node.mdx:13]: my editor auto fixed these

**Comment by jcstein** [comment]
do you think it is safe to provide the url over placeholder? assuming the user has already gone through the tutorial to set a node up, which would have showed them the example commands?

**Comment by rootulp** [comment]
> do you think it is safe to provide the url over placeholder?

IMO yes. I think the placeholders are less helpful than the commands with values populated. If a user needs to override a value, they can replace the populated value rather than the placeholder value. In fact, instead of repeating a command twice: once with placeholders and once with populated values, I think the docs would be easier to scan if there was only one instance per command with as many values populated as possible.

File: docs/nodes/light-node.mdx [hunk 2]
```diff
@@ -1101,7 +1101,7 @@ You will need to fund that address with testnet tokens to pay for
 <TabItem value="mocha" label="Mocha">
 
 ```sh
-celestia light start --core.ip <ip-address> --keyring.accname <key-name> --gateway.deprecated-endpoints --gateway --gateway.addr <ip-address> --gateway.port <port> --p2p.network mocha
+celestia light start --core.ip rpc-mocha.pops.one --keyring.accname <key-name> --gateway.deprecated-endpoints --gateway --gateway.addr <ip-address> --gateway.port <port> --p2p.network mocha
```
Comment (jcstein) [inline docs/nodes/light-node.mdx:1104]:
```suggestion
celestia light start --core.ip rpc-mocha.pops.one --keyring.accname my_celes_key --gateway.deprecated-endpoints --gateway --gateway.addr <ip-address> --gateway.port <port> --p2p.network mocha
```

File: docs/nodes/light-node.mdx [hunk 3]
```diff
@@ -1112,7 +1112,7 @@ You will need to fund that address with testnet tokens to pay for
 <TabItem value="arabica" label="Arabica 🏗️">
 
 ```sh
-celestia light start --keyring.accname <key_name> --core.ip <ip-address> --gateway.deprecated-endpoints --gateway --gateway.addr <ip-address> --gateway.port <port> --p2p.network arabica
+celestia light start --keyring.accname <key_name> --core.ip consensus-full-arabica-9.celestia-arabica.com --gateway.deprecated-endpoints --gateway --gateway.addr <ip-address> --gateway.port <port> --p2p.network arabica
```
Comment (jcstein) [inline docs/nodes/light-node.mdx:1115]:
```suggestion
celestia light start --keyring.accname <key_name> --core.ip consensus-full-arabica-9.celestia-arabica.com --p2p.network arabica
```

File: docs/nodes/light-node.mdx (hunk 1 above)
Comment (jcstein) [reply docs/nodes/light-node.mdx:13]: just will make sure it builds fine, should be good. thank you!

**Comment by jcstein** [comment]
as always, thank you for this PR!

**Review by jcstein — APPROVED** [review]

**Pull request merged**